README.md

```diff
@@ -2,179 +2,86 @@
 
  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![GoDoc](https://godoc.org/github.com/pravega/pravega-operator?status.svg)](https://godoc.org/github.com/pravega/pravega-operator) [![Build Status](https://travis-ci.org/pravega/pravega-operator.svg?branch=master)](https://travis-ci.org/pravega/pravega-operator) [![Go Report](https://goreportcard.com/badge/github.com/pravega/pravega-operator)](https://goreportcard.com/report/github.com/pravega/pravega-operator) [![Version](https://img.shields.io/github/release/pravega/pravega-operator.svg)](https://github.com/pravega/pravega-operator/releases)
 
-### Project status: alpha
-
-The project is currently alpha. While no breaking API changes are currently planned, we reserve the right to address bugs and change the API before the project is declared stable.
-
 ## Table of Contents
 
  * [Overview](#overview)
- * [Requirements](#requirements)
- * [Quickstart](#quickstart)    
- * [Install the Operator](#install-the-operator)
- * [Deploying in Test Mode](#deploying-in-test-mode)
- * [Upgrade the Operator](#upgrade-the-operator)
- * [Install a sample Pravega Cluster](#install-a-sample-pravega-cluster)
- * [Scale a Pravega Cluster](#scale-a-pravega-cluster)
- * [Upgrade a Pravega Cluster](#upgrade-a-pravega-cluster)
- * [Uninstall the Pravega Cluster](#uninstall-the-pravega-cluster)
- * [Uninstall the Operator](#uninstall-the-operator)
- * [Manual installation](#manual-installation)
- * [Configuration](#configuration)
+    * [Features](#features)
+    * [Project status](#project-status)
+    * [Requirements](#requirements)
+ * [Quickstart](#quickstart)
+    * [Install the Operator](#install-the-operator)
+    * [Install the Cluster](#install-the-cluster)
+    * [Installing on minikube](#installing-on-minikube)
+    * [Script for complete Pravega Installation](#script-for-complete-pravega-installation)
+    * [Scale the Cluster](#scale-the-cluster)
+    * [Upgrade the Cluster](#upgrade-the-cluster)
+    * [Upgrade the Operator](#upgrade-the-operator)
  * [Development](#development)
  * [Releases](#releases)
+ * [Community](#community)
  * [Troubleshooting](#troubleshooting)
 
 ## Overview
 
 [Pravega](http://pravega.io) is an open source distributed storage service implementing Streams. It offers Stream as the main primitive for the foundation of reliable storage systems: *a high-performance, durable, elastic, and unlimited append-only byte stream with strict ordering and consistency*.
 
-The Pravega Operator manages Pravega clusters deployed to Kubernetes and automates tasks related to operating a Pravega cluster.
+The Pravega Operator manages Pravega clusters deployed to Kubernetes and automates tasks related to operating a Pravega cluster.The operator itself is built with the [Operator framework](https://github.com/operator-framework/operator-sdk).
+
+### Features
```

Comment:
Each bullet point in this section should have links to documents that provide details about that feature, instead of having separate sections for "Installing Cluster", "Upgrading Cluster". The idea is to keep this document concise and push all the details into linked documents.